### PR TITLE
Provided support to manage record:Host objects without DNS

### PR DESCRIPTION
**Pull request by saiprasannasastry**
added a new method to get Network from reference
Two Get methods are written for Record Host
   one to Get with Ref(dependent on the ref passed by create method)  
   one to Get with necessary Parameters

### CODE REVIEW

**Comment by chinmayb** [comment]
Looks like johnson has already created a PR here : https://github.com/infobloxopen/infoblox-go-client/pull/52
add a new commit into that.  Dont create a new RP

**Comment by jkraj** [comment]
Hi @chinmayb @saiprasannasastry  have done some considerable amount of changes on top of my PR #52  we decide to submit as a new PR and abandon #52. 

@saiprasannasastry incorporate all the comments mentioned in #52 by both and @chinmayb and @yuewko.

**Comment by saiprasannasastry** [comment]
@jkraj  changed recordhost to hostrecord 
the methods i added on top of your's were only two methods.
I did Skype you about the change.

File: object_manager_test.go [hunk 1]
```diff
@@ -444,6 +568,33 @@ var _ = Describe("Object Manager", func() {
 		})
 	})
 
+	Describe("Get Network with Ref", func() {
```
Comment (jkraj) [inline object_manager_test.go]: ref => Reference

File: object_manager_test.go [hunk 2]
```diff
@@ -328,6 +330,128 @@ var _ = Describe("Object Manager", func() {
...
+		It("should pass expected host Address Object to CreateObject", func() {
+			actualRecord, err = objMgr.CreateHostRecordWithoutDNS(recordName, netviewName, cidr, ipAddr, macAddr, vmID)
+		})
+		It("should return expected host Address Object", func() {
+			Expect(actualRecord).To(Equal(aniFakeConnector.resultObject))
+			Expect(err).To(BeNil())
+		})
+	})
+
+	Describe("Allocate next available host Record", func() {
```
Comment (jkraj) [inline object_manager_test.go]: change the descriptions accordingly

File: object_manager_test.go [hunk 3]
```diff
@@ -593,6 +778,30 @@ var _ = Describe("Object Manager", func() {
 		})
 	})
 
+	Describe("Delete Host Address", func() {
```
Comment (jkraj) [inline object_manager_test.go]: address => Record

**Pull request merged**